e2e/aks_model.go

```diff
@@ -150,7 +150,14 @@ func airGapSecurityGroup(location, clusterFQDN string) (armnetwork.SecurityGroup
 func addPrivateEndpointForACR(ctx context.Context, t *testing.T, nodeResourceGroup string, vnet VNet) error {
 	t.Logf("Adding private endpoint for ACR in rg %s\n", nodeResourceGroup)
 
-	peResp, err := createPrivateEndpoint(ctx, t, nodeResourceGroup, vnet)
+	privateEndpointName := "PE-for-ABE2ETests"
+	if exists, err := privateEndpointExists(ctx, t, nodeResourceGroup, privateEndpointName); err != nil {
```

Comment:
```suggestion
	if exists, err := privateEndpointExists(ctx, t, nodeResourceGroup, privateEndpointName); err != nil {
	    return err
	}
	if exists {
	    return nil
	}
```

Reply:
this doesn't make the ide happy with 
exists declared and not used

can go with 
exists, err := privateEndpointExists(ctx, t, nodeResourceGroup, privateEndpointName)
	if err != nil {
		return err
	}
	if exists {
		t.Logf("Private Endpoint already exists, skipping creation")
		return nil
	}
	
if its about readability